app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -325,41 +326,67 @@ submitFeedback=Help Center
 swipeNavigationDistance=Swipe Navigation Distance
 switchToNewTabs=Switch to new tabs immediately
 sync=Sync
-syncAdd=I have an existing Sync code
-syncBetaMessage=Note: Sync for iOS and Android is still being completed and will be released soon.
+syncAddCode=Enter a sync code
+syncAddButton=Add
+syncAddDevice=Add device
 syncBookmarks=Bookmarks
-syncClearData=Clear Data
+syncChainCodeComputer=Now, using Brave on your computer, enter this code:
+syncChainCodeMobile=Now, using Brave on your mobile device, enter this code:
+syncChainCodeDescription=Go to: Brave Settings > Sync and click "Enter a Sync Chain Code"
+syncChooseDevice=Choose a device type:
 syncClearProfile=Sync a new device…
+syncConfirm=Confirm Sync Code
 syncCreate=Set up Sync
-syncData=Sync Data
+syncData=Data to Sync from {{device}}
 syncDataMessage=Sync the following data from this device:
+syncDeviceAddedOn=Added On
 syncDeviceLastActive=Last active
-syncDeviceName=Device name
+syncDeviceLoading=Loading devices. Please wait...
+syncDeviceListLoaded=Great! Your devices are now in sync
+syncDeviceListLoading=Updating devices list
+syncDeviceName=Device Name
 syncDeviceNameInput=Enter an optional name for this device:
 syncDevices=Devices
-syncEnable=Sync this device
-syncEnterPassphrase=Enter your Sync code words:
+syncDevicesDescription=Your synced devices are listed below. You can add additional devices now or at any time in the future.
+thisDevice=(this device)
+syncDevicesInSyncChain=Devices in your sync chain
+syncEnterPassphrase=Go to: Brave Settings > Sync > Display sync code
 syncHidePassphrase=Hide code words
 syncHideQR=Hide QR code
 syncHistory=Browsing history
+syncLeaveChainButton=Leave Sync Chain
+syncLookingForDevice=Looking for device
+syncNewDeviceTitle=Let's sync a new device with "{{device}}"
 syncNewDevice1=Open Brave on your new device and go to Preferences > Sync > 'I have an existing synced device'.
 syncNewDevice2=If it asks you to scan a QR code, click the button below and point your camera at the QR code.
 syncNewDevice3=If asks you to enter code words, type in the words below.
-syncNewDevice=Sync a new device…
+syncNewDeviceButton=Sync a new device…
+syncPhoneOrTablet=Phone/Tablet
+syncQRCode=QR Code
 syncQRImg.title=Brave sync QR code
+syncRemove=Remove
+syncRemoveDeviceModal=Remove "{{device}}" from this sync chain?
+syncRemoveDevice.title=Remove this device
+syncRemoveActiveDeviceWarning1=Local device data will remain intact on all devices. Other devices in this sync chain will remain sync'd.
```

Comment:
can we use `synced` instead of `sync'd` (which isn't a word)?

Reply:
I hate the way the word looks but since it is more correct, let's use "synced".

Reply:
done in 63c57cd76e472929fefdc66b030c088b7c87f4b3